README.md

```diff
@@ -51,8 +43,6 @@ Installs and configures Podman.
 | __default_runtime__  | string | `crun` | The default container runtime to use for Podman. |
 | __default_cgroup_manager__  | string | `cgroupfs` | The default cgroup manager to use for Podman. |
 
-<br>
-
 ## 4.2. Setup
 
 Installs and configures a Receptor node.
```

Comment:
These three lines are currently rendered as one long line

So you want
```
Installs and configures a Receptor node.

* On Centos/Redhat/Fedora it can be used the dnf package already bundled
* On Debian you can set `receptor_install_method` to `release` in that way we can download pre-build package created by GitHub Action